securing_apps/topics/saml/java/jboss-adapter/jboss-adapter-samesite-setting.adoc

```diff
@@ -0,0 +1,17 @@
+[[_saml-jboss-adapter-samesite-setting]]
+===== Setting SameSite value for JSESSIONID cookie
+
+Browsers are planning to set default value for the `SameSite` attribute for cookies to `Lax`. This means that
```

Comment:
Add "the" before "the"
Try to always follow "this" with a noun, so it is clear what "this" refers to.   "This setting"